Gems/MiniAudio/Code/Platform/Mac/platform_mac.cmake

```diff
@@ -0,0 +1,7 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
```

Comment:
do we not also need the same fixes for Mac that iOS has?

Reply:
turns out we do not. Per the documentation in miniaudio: https://miniaud.io/docs/manual/index.html#Building

miniaudio should compile just fine for mac OS on its own. I also tested this in Xcode when building the tools target as well.